smithy-model/src/test/java/software/amazon/smithy/model/validation/suppressions/TraitSuppressionTest.java

```diff
@@ -0,0 +1,66 @@
+package software.amazon.smithy.model.validation.suppressions;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import software.amazon.smithy.model.shapes.Shape;
+import software.amazon.smithy.model.shapes.StringShape;
+import software.amazon.smithy.model.traits.SuppressTrait;
+import software.amazon.smithy.model.validation.Severity;
+import software.amazon.smithy.model.validation.ValidationEvent;
+import software.amazon.smithy.utils.ListUtils;
+
+public class TraitSuppressionTest {
+
+    @Test
+    public void doesNotMatchWhenUsingDifferentShape() {
+        Shape s = StringShape.builder()
+                .id("smithy.example#String")
+                .addTrait(SuppressTrait.builder().values(ListUtils.of("Foo")).build())
+                .build();
+        Suppression suppression = Suppression.fromSuppressTrait(s);
+        ValidationEvent event = ValidationEvent.builder()
+                .id("Foo")
```

Comment:
Technically correct in that _no_ shape on the event is different from `s`, but clearer with a specified shape.